### PR TITLE
Change the version to 4.8.0

### PR DESCRIPTION
**Pull request by nkolev92**
Bumping up the build version.

Changing the CLI/SDK versions to 2.1.3xx.

### CODE REVIEW

File: build/config.props [hunk 1]
```diff
@@ -4,16 +4,16 @@
...
-    <VsTargetBranch Condition="'$(IsEscrowMode)' == 'true'">rel/d15.$(MinorNuGetVersion)</VsTargetBranch>
-    <SdkTargetBranch>release/2.1.2xx</SdkTargetBranch>
-    <CliTargetBranch>release/2.1.2xx</CliTargetBranch>
+    <VsTargetBranch Condition="'$(IsEscrowMode)' == 'false'">rel/d15.$(MinorNuGetVersion)</VsTargetBranch>
+    <SdkTargetBranch>release/2.1.4xx</SdkTargetBranch>
+    <CliTargetBranch>release/2.1.4xx</CliTargetBranch>
     <!-- We need to update this netcoreassembly build number with every milestone to workaround any breaking api
     changes we might have made-->
-    <NetCoreAssemblyBuildNumber Condition=" '$(NetCoreAssemblyBuildNumber)' == '' ">6</NetCoreAssemblyBuildNumber>
+    <NetCoreAssemblyBuildNumber Condition=" '$(NetCoreAssemblyBuildNumber)' == '' ">0</NetCoreAssemblyBuildNumber>
```
Comment (rohit21agrawal) [inline build/config.props]: this should be 1 , since we want preview1 packages

File: build/config.props [hunk 2]
```diff
@@ -4,16 +4,16 @@
...
     <MajorNuGetVersion Condition="'$(MajorNuGetVersion)' == ''">4</MajorNuGetVersion>
-    <MinorNuGetVersion Condition="'$(MinorNuGetVersion)' == ''">7</MinorNuGetVersion>
+    <MinorNuGetVersion Condition="'$(MinorNuGetVersion)' == ''">8</MinorNuGetVersion>
     <PatchNuGetVersion Condition="'$(PatchNuGetVersion)' == ''">0</PatchNuGetVersion>
     <SemanticVersion Condition=" '$(SemanticVersion)' == '' ">$(MajorNuGetVersion).$(MinorNuGetVersion).$(PatchNuGetVersion)</SemanticVersion>
     <VsTargetBranch>lab/d15.$(MinorNuGetVersion)stg</VsTargetBranch>
-    <VsTargetBranch Condition="'$(IsEscrowMode)' == 'true'">rel/d15.$(MinorNuGetVersion)</VsTargetBranch>
-    <SdkTargetBranch>release/2.1.2xx</SdkTargetBranch>
-    <CliTargetBranch>release/2.1.2xx</CliTargetBranch>
+    <VsTargetBranch Condition="'$(IsEscrowMode)' == 'false'">rel/d15.$(MinorNuGetVersion)</VsTargetBranch>
```
Comment (rohit21agrawal) [inline build/config.props]: you want to change the property value, not the condition

File: build/config.props [hunk 3]
```diff
@@ -4,16 +4,16 @@
...
+    <SdkTargetBranch>release/2.1.4xx</SdkTargetBranch>
```
Comment (rohit21agrawal) [inline build/config.props]: should be 2.1.3xx

**Comment by nkolev92** [comment]
@rohit21agrawal 
Addressed all the feedback

**Review by rohit21agrawal — APPROVED** [review]

**Review by jainaashish — APPROVED** [review]

**Pull request merged**